### ISSUE TITLE
[FEATURE REQUEST] Dark-mode compatible SVGs.

### ISSUE THREAD

**Issue by Thomas-Boi** [devicons/devicon#977]
### Problem

A few users wanted dark-mode compatible versions of SVGs currently in the repo (ex. see #973). 

### Suggested feature

We could allow dark-mode compatible SVGs to be added to the repo. Before, we weren't accepting them since it'll clutter the repo. If the user wants dark-mode, they can just download the SVGs and edit it to their taste. However, more and more people are now using dark-mode. Perhaps we should accommodate these needs from the community and change our policy.

### Additional information

This would require:
- A new naming scheme for dark-mode compatible SVGs. I'm thinking it's something optional, like `wordmark`. So say an SVG can be named `express-original-dark.svg` or something similar.
- Updates to the docs about this new type of icons
- Updates to the websites and script checking to accommodate this new icon type.

**Comment by Thomas-Boi** [comment]
After some thoughts, I think having an extra tag for dark-mode friendly SVGs would be unnecessary and could be confusing. Say we have a normal icon and a dark-mode compatible icon. Cool, we add `-dark` to the second icon. However, what if the `original` icon was already dark-mode compatible? Then, do we make a `-light` version of the icon that looks good in light mode? Do we not do it at all?

I'm thinking it'd be better to just change the color using an API (see #913) rather than having an SVG in the repo.

**Comment by WolfgangVing** [comment]
> I'm thinking it'd be better to just change the color using an API (see #913) rather than having an SVG in the repo.

I agree this API/"CND" would make a difference for users like me who love dark themes and of course for developers, I think what I mentioned about the [github-profile-readme-generator](https://github.com/rahuldkjain/github-profile-readme-generator) can help resolve the question of being welcomed, this tool has been cited in plenty of articles. I am sure there will be many more who will use your content indirectly.
